lib/screens/qpi_page.dart

```diff
@@ -1,15 +1,113 @@
+import 'package:admu_student_app/models/year.dart';
```

Comment:
pls follow the imports format (so that our code is clean)

```dart
dart imports
// space
other package imports
// space
project imports
```
(also in alphabetical order)